src/sonic-yang-models/yang-models/sonic-flex_counter.yang

```diff
@@ -327,6 +327,19 @@ module sonic-flex_counter {
                 }
             }
 
+            container SRV6 {
+                /* SRV6_STAT_COUNTER_FLEX_COUNTER_GROUP */
+                leaf FLEX_COUNTER_STATUS {
+                    type flex_status;
+                }
+                leaf FLEX_COUNTER_DELAY_STATUS {
```

Comment:
@stepanblyschak 
I remember you have removed the `flex_delay_status` field. Do we still need it in the yang model here?
or maybe we can have it for now just like all other counter groups
and then have another PR to remove it